plugins/item_tasks/plugin_tests/tasks_test.py

```diff
@@ -103,12 +103,16 @@ def testAddItemTasksToFolderFromJson(self):
         with open(os.path.join(os.path.dirname(__file__), 'spec.json')) as f:
             spec = f.read()
         parsedSpec = json.loads(spec)
+
+        token = self.model('token').createToken(
+            user=self.admin, scope='item_task.set_task_spec.%s' % folder2['_id'])
```

Comment:
Makes sense. Would it be good to negative test as well by, say, sending a request without a token?